.github/workflows/go.yml

```diff
@@ -20,8 +20,10 @@ jobs:
         uses: actions/setup-go@v3
```

Comment:
There is no real change in this file. Can you please revert?

Reply:
I changed this file as it is the fix of lint. (I couldnt pass it)